src/TextArea.tsx

```diff
@@ -0,0 +1,10 @@
+import tw from 'twin.macro'
+
+const TextArea = tw.textarea`
```

Comment:
Would this file make more sense in `/src/style`?

Reply:
> * It looks like the Dark/Light theming is lost in this PR. What is your strategy for theming?

For themes, we are using the `tailwind` approach here. https://github.com/shresthabijay/em/blob/cfe81de2bd77a30f96bdc61c246aa173fdfdf853/src/components/ActionButton.tsx#L29-L32

It is compatible with the current theme implementation. If `.dark` class is added to the body, the `dark` variant of the `tailwind` will be applied. 

You can see how `tailwind` handles it https://tailwindcss.com/docs/dark-mode.